e2e/__tests__/randomize.test.ts

```diff
@@ -0,0 +1,47 @@
+import * as path from 'path';
```

Comment:
Minor detail. Could you run `yarn check-copyright-headers` and headers to file which miss them. Just copy it from any file: https://github.com/facebook/jest/blob/5e1e105e64ab68415e1549a0f16044d888ef3caf/e2e/__tests__/browserResolve.ts#L1-L8